teams/templates/includes/empty_project.html

```diff
@@ -0,0 +1,23 @@
+{% load static %}
+
+<div class="row mt-3 mb-2 mt-4 ml-4"> 
+    <div class="col">
+        <h3>Project: {{ team.project.display_name}}</h3>
+    </div>
+</div>
+
+<div class="row mt-3 mb-5 ml-4 mr-4"> 
+    <div class="col">
+        <div class="card p-3">
+            <div class="card-body">
+                No Project submitted yet.
+                {% if request.user in team.participants.all %} 
+                <a href="/teams/{{team.id}}/project" class="teams-link">Add project details now!</a>
+                <p class="empty-project-subtitle mt-2 mb-0">
+                    <i>Project details can always be edited until the judging starts.</i>
+                {% endif %}
+                </p>
+            </div>
+        </div>
+    </div>
+</div>
```

Comment:
Missing blank line at the end of the file.

Reply:
Added blank line.